boot/zephyr/Kconfig

```diff
@@ -782,4 +786,13 @@ config MCUBOOT_VERIFY_IMG_ADDRESS
 	  also be useful when BOOT_DIRECT_XIP is enabled, to ensure that the image
 	  linked at the correct address is loaded.
 
+# When used with MCUboot, the external flash page size must match
+# the internal SoC flash page size, which is 4096 for Nordic MCUs.
+configdefault NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE
```

Comment:
Can NORDIC specific configuration moved from the generic to nordic-specific files?

Reply:
Yeah maybe this could moved into a specific Kconfig.nrf file which is included when using a nRF file?